CONTRIBUTING.md

```diff
@@ -49,7 +49,7 @@ An easy way of testing changes to the oozie-to-airflow converter is by using  Cl
 
 #### Setup
 1. Create a [Dataproc cluster with Oozie initialization actions.](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/tree/master/oozie#oozie)
-1. Run the `init.sh` shell script to install required python dependencies on the dev machine
+1. Run the `setup.py` script to install required python dependencies on the dev machine
 1. Install other required depedencies, for example, Airflow's `SparkSubmitOperator` requires `spark-submit` installed on the cluster.
```

Comment:
Is init.sh going to be used anymore? Should we delete it?

Reply:
I just opened a requirements.txt not knowing that this PR is in progress https://github.com/GoogleCloudPlatform/cloud-composer/pull/9 - maybe we should opt to use requirements.txt for now rather than using setup.py until we really need more complex setup ? Just a thought. I am fine with either approach